source/query/recent.txt

```diff
@@ -0,0 +1,24 @@
+.. _recent-queries:
+
+===================
+View Recent Queries
+===================
+
+.. default-domain:: mongodb
+
+You can view up to 20 of the most recent queries for that collection.
+Click a listed query to populate the query bar with that query.
+
+.. figure:: /images/compass/query-history-select.png
+
+You can also hover over a query and perform the following actions:
+
+- Click the :guilabel:`Star` icon to save the query as a
```

Comment:
Should we be using the image of the icon versus a verbal description "Star"? I noticed we used images on the other pages, so I thought it would make sense to be consistent.

Reply:
Agreed. Fixed.

Reply:
Damn - that looks good!